packages/core/content-manager/server/services/entity-manager.js

```diff
@@ -39,24 +39,44 @@ const findCreatorRoles = (entity) => {
   return [];
 };
 
-// TODO: define when we use this one vs basic populate
-const getDeepPopulate = (uid, populate) => {
+const getDeepPopulate = (
+  uid,
+  populate,
+  { onlyMany = false, countMany = false, maxLevel = Infinity } = {},
```

Comment:
In which case would you limit the level of this populate?

Reply:
Should we rename maxLevel to depth?